development/releasing.md

```diff
@@ -2,29 +2,42 @@
 
 ## Webapp
 
-When all the required issues are fixed in master it's time to put together a release.
+### Planning
 
-1. Pick a version number. We use [semver](http://semver.org) so if there are breaking changes increment the major, otherwise if there are new features increment the minor, otherwise increment the service pack. Breaking changes in our case relate to updated software requirements (egs: CouchDB, node, minimum browser versions) or broken backwards compatibility in an api.
-2. If releasing a new major or minor...
-  - Update [google-libphonenumber](https://www.npmjs.com/package/google-libphonenumber) to the latest version in api, sentinel, and webapp and commit to `master` and push. This ensures we're up to date with the latest in phone number validation.
-```bash
-npm install --save google-libphonenumber@latest
+1. Create an [organisation wide project](https://github.com/orgs/medic/projects?query=is%3Aopen+sort%3Aname-asc). We use [semver](http://semver.org) so if there are breaking changes increment the major, otherwise if there are new features increment the minor, otherwise increment the service pack. Breaking changes in our case relate to updated software requirements (egs: CouchDB, node, minimum browser versions) or broken backwards compatibility in an api.
```

Comment:
We have been talking recently about major visual changes also being a "breaking change". This hasn't been finalized yet, but worth adding a note here that a breaking change could be anything that can't be upgraded to without large amount of effort from a deployment team. We could give the example of anything requiring retraining of users, such as a large visual change.